### PR TITLE
first proposed draft of glossary reset

### PR DESCRIPTION
**Pull request by jeremywalter**
removed bullets, added line separation lines, alphabetized

things that can be done:

- [ ] put bullets back

- [ ] remove separation lines

- [ ] move all links to definitions (not in the actual entry)

next steps: compare/add definitions from alternate glossary, edit/remove repeated content, massage entries, check/add links

### CODE REVIEW

**Comment by ACharbonneau** [comment]
I'm not sure what this means: 
> move all links to definitions (not in the actual entry)

File: docs/spec-and-docs/NewGlossary.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+##### <u>eRA Commons</u>
+the [`electronic Research Administration`](https://public.era.nih.gov/commons) is an online interface where signing officials, principal investigators, trainees, and post-docs at institutions/organizations can access and share administrative information relating to research grants. It is the `designated ID provider` for the whitelist of DCCs.
+
+##### [FAIR](https://www.go-fair.org/fair-principles/)
+`Findable, Accessible, Interoperable, and Reusable`. Making data FAIR is the main goal of the DCCs.
+
+##### <u>FAIR Assessment</u>
+a process which evaluates the level of compliance of a dataset with the FAIR principles. The assessment may be performed with tools such as [FAIRshake](https://fairshake.cloud) or [FAIREvaluator](https://fairsharing.github.io/FAIR-Evaluator-FrontEnd/#!/).
+
+##### [FAIR Insignia]()
```
Comment (ACharbonneau) [inline docs/spec-and-docs/NewGlossary.md:217]: broken link

File: docs/spec-and-docs/NewGlossary.md [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+##### <u>File</u>
+a `digital asset`, that is a type of digital object that each of the [DCCs](#DCC) hosts, such as genomic, metagenomic, RNA-Seq sequence data, physiological, and metabolic data or generic metadata electronic document.
+
+##### [Frictionless Data Package](http://frictionlessdata.io/data-package/)
+a format specification produced by the [Open Knowledge Foundation](https://okfn.org/) and supported by the Frictionlessdata.io organization. It aims to shorten the path from data to insight with a collection of specifications and software for the publication, transport, and consumption of data. This kills the cycle of find/improve/share that makes for a dynamic and productive data ecosystem.
+
+##### [GitHub](https://github.com/)
+an online hub for storing and sharing computer programs and other plain text files. The [CFDE](#CFDE) team uses it for storage, hosting websites, communications, and project management.
+
+##### [Globus Automate]()
```
Comment (ACharbonneau) [inline docs/spec-and-docs/NewGlossary.md:237]: broken link

Comment (jeremywalter) [reply docs/spec-and-docs/NewGlossary.md:237]: I didn't test the links yet... this is purely an aesthetic/structural check-in to make sure this is moving in the right direction. At the bottom are what I believe are a bunch of things that need definitions and places in the list left by someone in the past. If the format and set-up looks okay, then the pruning can begin.

The link question: some of the entries in the glossary are hyperlinked themselves, some of the hyperlinks are in the definition instead... I think one convention should be followed but I would not love to 'choose' the wrong one and go change them all back. Right now it's a totally mixed bag.

Comment (ACharbonneau) [reply docs/spec-and-docs/NewGlossary.md:237]: ahh. format is fine. 

I think the ones that are hyperlinks are links out to other websites? I don't have an opinion on that. whatever works.

**Comment by jeremywalter** [comment]
since this isn't going live, can we merge (create) the file just to clean up the commits a bit?

**Comment by jeremywalter** [comment]
the glossary has been updated and the 'extra' file has been removed... some definitions are missing and help is needed to fill them in... glossary is now A-Z with added content from the other glossary file in this repo and use case definitions deprecated

**Comment by jeremywalter** [comment]
Though there are many incomplete (I think) definitions in the glossary right now... these entries are currently blank:

**Association Table
Associative Relationship
C2M2 Instance
Foreign Key
Serialization
Subject Granularity
Subject Role**

**Comment by ACharbonneau** [comment]
Are those all words from arthurs doc?

**Comment by jeremywalter** [comment]
they are a mix...arthur words, and mishmash from the bottom of the glossary

**Comment by ACharbonneau** [comment]
If the words are used somewhere we don't need them. I know the Arthur words are curated, did you look for the others?

**Comment by jeremywalter** [comment]
> If the words aren't used somewhere we don't need them. I know the Arthur words are curated, did you look for the others?

I have seen mentions of them, but in the context of... gosh... what does that mean... go to glossary... blank. Marisa's link is a good example... where I went to look at Association Table and it was not something I could parse as an average human.

**Comment by ACharbonneau** [comment]
I don't really think I followed your last comment, but the words from Arthur I know are in the docs and need definitions. But we can just...ask Arthur for them? I'm asking about the ones that aren't from Arthur. Do we actually need those words? 

Also, can I have the list of Arthur words you want defined?

**Comment by marisalim** [comment]
Marisa's link was the [C2M2 level documentation](https://github.com/nih-cfde/specifications-and-documentation/blob/master/draft-C2M2_specification_with_Levels/README.md) and it is true that it uses the words but doesn't have nice obvious definitions. ;)

**Comment by jeremywalter** [comment]
> I don't really think I followed your last comment, but the words from Arthur I know are in the docs and need definitions. But we can just...ask Arthur for them? I'm asking about the ones that aren't from Arthur. Do we actually need those words?
> 
> Also, can I have the list of Arthur words you want defined?

Looks like the words I listed are all Arthur words (I found definitions for others)

**Comment by ACharbonneau** [comment]
@abradyIGS reports that he can have us those definitions this week :)

**Comment by jeremywalter** [comment]
Missing definitions: 
**Collection
Identifier**

**Comment by ACharbonneau** [comment]
Still needs work, but is much better than the current display so we're gonna merge and continue fixing up from there

**Pull request merged**